app/src/main/java/io/neurolab/fragments/MemoryGraphFragment.java

```diff
@@ -315,14 +318,19 @@ public boolean onOptionsItemSelected(MenuItem item) {
         if (id == R.id.import_data) {
             getRuntimePermissions();
             selectCSVFile();
-        } else if(id == R.id.info_program) {
+        } else if (id == R.id.info_program) {
             AlertDialog.Builder progress = new AlertDialog.Builder(getContext());
             progress.setCancelable(true);
             progress.setTitle(R.string.program_info_label);
             progress.setMessage(R.string.program_info);
             AlertDialog infoDialog = progress.create();
             infoDialog.show();
 
+        } else if (id == R.id.stop_data) {
+            parsedData = null;
+            Toast.makeText(getContext(), "Stopped", Toast.LENGTH_SHORT).show();
+            initializeMemGraph(memGraph);
```

Comment:
Stopping should display the data right? But it seems to clear the whole graph :thinking:

Reply:
Okay so are you saying to stop the graph in it's last/latest position without completely clearing it?

Reply:
Yes

Reply:
Cool. will do that then. I thought that would be more appropriate as a pause feature.